githubAPI/octokit.js

```diff
@@ -0,0 +1,58 @@
+const { Octokit } = require("@octokit/rest");
+require('dotenv').config();
+
+const octokit = new Octokit({
+  auth: process.env.WEBHOOK_SECRET,
+  log: {
+    debug: () => {},
+    info: () => {},
+    warn: console.warn,
+    error: console.error
+  }
+});
+
+if(!octokit.auth) {
+  console.log('Missing authentication token. Please add correct environment variable to file .env!');
+  return undefined;
+}
+
+module.exports.asyncTryToken = async function() {
```

Comment:
As above wrt. removing globals, I think would be good to have this function called with the token, e.g.:

```suggestion
const isValidToken = async (token) => {
```